sktime/annotation/hmm.py

```diff
@@ -374,26 +374,25 @@ def _predict(self, X):
             Array of predicted class labels, same size as input.
         """
         self.num_states = len(self.emission_funcs)
-        self.states = [i for i in range(self.num_states)]
+        self.states = list(range(self.num_states))
         self.num_obs = len(X)
         emi_probs = HMM._make_emission_probs(self.emission_funcs, X)
         init_probs = self.initial_probs
-        # if no initial_probs were supplied assign all states equal prob:
         if self.initial_probs is None:
-            init_probs = 1.0 / (self.num_states) * np.ones(self.num_states)
+            init_probs = 1.0 / self.num_states * np.ones(self.num_states)
         trans_prob, trans_id = HMM._calculate_trans_mats(
```

Comment:
FYI since you are using `@staticmethod` you can do
```suggestion
        trans_prob, trans_id = self._calculate_trans_mats(
```

Reply:
Nice :+1 I am not really sure what the difference is between these two things, but seems reasonable to me.  Will look more into it, but in the meantime went ahead and make the change! :)